### DISCUSSION TITLE
Where is the xpi to download????

### DISCUSSION THREAD

**Discussion by rebop** [onemen/TabMixPlus#161] [General]
Looked for ever and cannot find it. Can you show me clearly where it is found?

Thanks.

**Comment by onemen** [comment]
[Tab Mix Plus - README](https://github.com/onemen/TabMixPlus#readme)
[Releases](https://github.com/onemen/TabMixPlus/releases)

![releases](https://user-images.githubusercontent.com/3650909/188059383-c0111a37-3aed-46c6-8390-81aa40db8ecb.png)
![assets](https://user-images.githubusercontent.com/3650909/188059390-0f223bb2-4bc7-4855-9293-88d48593478b.png)

**Comment by rebop** [comment]
I did EXACTLY that many times. When I click on the .xpi it tries to install and then says corrupt. I can find no way to download it.

Thanks.

**Comment by rebop** [comment]
Got it. I had missed a step in in the install process of the UserChromeJS. Since I am testing this on a prtable version, might be helpful to have instructions for that situation specifically. And I also have a Chrome folder with many thing from https://github.com/Aris-t2/CustomCSSforFx. But it seems to be working now.
